Resources/Prototypes/Entities/Objects/Weapons/Melee/e_sword.yml

```
@@ -0,0 +1,18 @@
+- type: entity
+  name: energy sword
+  parent: BaseItem
+  id: EnergySword
+  description: A very dangerous energy sword. Can be stored in pockets when turned off. Makes a lot of noise when used or turned on.
+  components:
+  - type: Esword
+  - type: Sprite
+    sprite: Objects/Weapons/Melee/e_sword.rsi
+    state: e_sword
+  - type: MeleeWeapon
+    damage:
+      types:
+        Blunt: 5
+  - type: Item
+    size: 5
+    sprite: Objects/Weapons/Melee/e_sword.rsi
+    
```

Should have a use delay to prevent people from spamming the on/off sound effect.
```suggestion
    sprite: Objects/Weapons/Melee/e_sword.rsi
    - type: UseDelay
```